examples/chains/neutron_gaia.json

```
@@ -0,0 +1,206 @@
+{
+  "chains": [
+    {
+      "name": "gaia",
+      "chain_id": "localcosmos-1",
+      "denom": "uatom",
+      "binary": "gaiad",
+      "bech32_prefix": "cosmos",
+      "docker_image": {
+        "version": "v15.0.0-rc2"
+      },
+      "gas_prices": "0%DENOM%",
+      "chain_type": "cosmos",
+      "coin_type": 118,
+      "trusting_period": "336h",
+      "gas_adjustment": 2.0,
+      "number_vals": 1,
+      "number_node": 0,
+      "ibc_paths": [],
+      "debugging": true,
+      "block_time": "500ms",
+      "host_port_override": {
+        "26657": "26658",
+        "1317": "1318",
+        "9090": "9091"
+      },
+      "genesis": {
+        "modify": [
+          {
+            "key": "app_state.gov.params.voting_period",
+            "value": "3s"
+          },
+          {
+            "key": "app_state.interchainaccounts.host_genesis_state.params.allow_messages",
+            "value": [
+              "/cosmos.bank.v1beta1.MsgSend",
+              "/cosmos.bank.v1beta1.MsgMultiSend",
+              "/cosmos.staking.v1beta1.MsgDelegate",
+              "/cosmos.staking.v1beta1.MsgUndelegate",
+              "/cosmos.staking.v1beta1.MsgBeginRedelegate",
+              "/cosmos.staking.v1beta1.MsgRedeemTokensforShares",
+              "/cosmos.staking.v1beta1.MsgTokenizeShares",
+              "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
+              "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress",
+              "/ibc.applications.transfer.v1.MsgTransfer"
+            ]
+          }
+        ],
+        "accounts": [
+          {
+            "name": "acc0",
+            "address": "cosmos1kuf2kxwuv2p8k3gnpja7mzf05zvep0cycmheu0",
```

let's stick to the accounts that are specified in current covenant e2e tests to avoid adjusting the existing tests. mnemonic will also need to be adjusted.
```suggestion
            "address": "cosmos1hj5fveer5cjtn4wd6wstzugjfdxzl0xpxvjjvr",
```

---

Implemented this. All hardcoded addrs including in chain config are consistent with the mnemonic in the covenants chain config.